src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+
+
 public class Duke {
+
+    private static final String filePath = "data/duke.txt";
+
+    /*private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }*/
+
+    private static void appendToFile(String filePath, String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true); // create a FileWriter in append mode
+        fw.write(textToAppend);
+        fw.close();
+    }
+
     public static void main(String[] args) {
+
+        File f = new File(filePath);
+        File folder = new File("data");
+        if (!folder.exists()) {
+            folder.mkdir();
+        }
+        if (!f.exists()) {
+            try {
+                f.createNewFile();
+            } catch (IOException e) {
+                System.out.println("Failed to create file to save tasks.");
+            }
+        }
+
+        System.out.println("full path: " + f.getAbsolutePath());
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+        //System.out.println("Hello from\n" + logo);
+        String GREET = "Hello! I'm Duke\n" + "What can I do for you?\n";
+        String EXIT = "Bye. Hope to see you again soon!\n";
+        String BYE = "bye";
+        String LIST = "list";
+        String MARK = "mark";
+        String TODO = "todo";
+        String DEADLINE = "deadline";
+        String EVENT = "event";
+        String DELETE = "delete";
+
+        System.out.println(GREET);
+
+        ArrayList<String> instructionsList = new ArrayList<>();
+
+        Task task = new Task("hello");
+        Task[] tasks = new Task[100];
+        task.number = 0;
+
+        Deadline deadline = new Deadline("return book", "holi");
+        Event event = new Event("test", "7pm");
+
+        for (int i = 0; i < 200; i++) { // can have 200 input lines (including wrong command)
+            Scanner in = new Scanner(System.in);
+            //System.out.print("Type something: ");
+            task.instruction = in.nextLine();
+            String instructionLine = task.instruction.replaceAll("todo|deadline|event", "");
+            String updatedInstructionLine; //updated instruction line is in the form of [][] instructionline
+
+            Task t = new Task("read book");
+            String[] arrOfStr = task.instruction.split(" ", 50);
+            String[] arrOfDeadline = instructionLine.split("/by ", 2);
+            String[] arrOfEvent = instructionLine.split("/at ", 2);
```

Comment:
Consider renaming these attributes. Replacing `arr` to `array` should prevent potential confusion, or perhaps renaming the entire names altogether.